### ISSUE TITLE
Hepatic infarction

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6164]
**Preferred term label:**
Hepatic infarction

**Synonyms**
Liver infarction, Hepatic infarct

**Definition (free text, please give PubMed ID)**
A very rare occurrence due to the multiple-source blood supply of the liver.
DOI: 10.5430/crim.v5n2p28 

**Parent term (use hpo.jax.org/app)**
Hepatic failure

**Comment by pnrobinson** [comment]
adding with def
Hepatic infarction is defined as areas of coagulative necrosis from hepatocyte cell death caused by local ischemia which, in turn, results from the obstruction of circulation to the affected area, most commonly by a thrombus or embolus. 

I think this should be a child of Abn. hepatic morphology, since hepatic failure is a possible consequence but not really the superclass
